packages/components/src/components/loading/dtrader-v2-loaders/positions-loader.tsx

```diff
@@ -17,7 +17,7 @@ const PositionsLoader = ({ initial_app_loading, is_closed_tab }: TPositionsLoade
                 </div>
             )}
             <div className='skeleton-box__filters'>
-                {[...new Array(is_closed_tab ? 2 : 1)].map((_, idx) => {
+                {[...new Array(1)].map((_, idx) => {
```

Comment:
We have array of 1 entry only . Can we remove the map and just return
`<Skeleton  width={144} height={32} />`